libafl_frida/harness.cpp

```diff
@@ -0,0 +1,63 @@
+#include <stdint.h>
```

Comment:
Maybe name this test_harness.cpp or something else that makes it clear this is not part of the actual library (or even move to a tests folder?)

Reply:
Renamed to test_harness